### DISCUSSION TITLE
version 0.25.0 of httpx (AsyncClient) leading to httpx.ReadTimeout

### DISCUSSION THREAD

**Discussion by SarloAkrobata** [encode/httpx#2932] [Potential Issue]
Hi, I have project which download a lot images from different websites, with a lot concurent requests using AsyncClient. When I upgraded from 0.23.3 to latest, after some period it was starting to produce httpx.ReadTimeout error. Does anyone have similar problem, or know where problem can be?

**Comment by dhensen** [comment]
Have you tried raising the timeout(s)? httpx is different from say requests in that it provide default timeouts. Those default timeouts might be too strict for you.

**Reply by SarloAkrobata** [reply]
Its not the case, it is working fine with 0.23.3.

**Comment by SarloAkrobata** [comment]
Probably it is related to this: https://github.com/agronholm/anyio/discussions/580
